config/config.go

```diff
@@ -464,7 +469,13 @@ func (ng *NetworkGroups) UnmarshalYAML(unmarshal func(interface{}) error) error
 	if err := unmarshal((*plain)(ng)); err != nil {
 		return err
 	}
-	return checkOverflow(ng.XXX, "network_groups")
+	if len(ng.Name) == 0 {
```

Comment:
Probably, this change should be pushed in a separate commit if it is correct, since it is unrelated to the PR.

Reply:
True. Should be fixed in a separate PR